Dockerfile

```
@@ -9,7 +9,8 @@ COPY pkg pkg
 COPY Makefile Makefile
 RUN make all
 
-FROM alpine:3.16.0
+FROM scratch
 USER 1000
```

Since we're cleaning up the Dockerfile, can we please also add a group here? (And maybe use 10k as a user, this is a nitpick, but a lesser chance of clashing with actual users on any system)

```suggestion
USER 10000:10000
```

---

but the user only exists in the docker image? Unless I am missing something?

---

happy to make that change

---

👍 and not really, while Docker supports users ns for some time, I believe these are not enabled by default [^1], i.e. user/gids are shared. I.e. it's a good security practice to avoid running as root, and other low numbers that are likely to exist on the host system.

[^1]: https://docs.docker.com/engine/security/userns-remap/